database/bigquery/copy_from_csv.py

```diff
@@ -0,0 +1,121 @@
+import os
+import json
+import tempfile
+import argparse
+
+import pandas as pd
+
+from google.cloud import bigquery
+from google.oauth2 import service_account
+from google.api_core.exceptions import NotFound
+
+
+def get_args():
+    parser = argparse.ArgumentParser()
+    parser.add_argument('--dataset', dest='dataset', required=True)
+    parser.add_argument('--table', dest='table', required=True)
+    parser.add_argument('--service-account', dest='service_account', required=True)
+    parser.add_argument('--source-file-name', dest='source_file_path',
+            default='output.csv', required=True)
```

Comment:
Default should be None, or whatever we've done in previous scripts. An input file shouldn't have a default name.